control_plane/src/compute.rs

```diff
@@ -279,20 +288,14 @@ impl PostgresNode {
             },
         )?;
 
-        let mut copyreader = client
-            .copy_out(sql.as_str())
-            .with_context(|| "page server 'basebackup' command failed")?;
-
-        // FIXME: Currently, we slurp the whole tarball into memory, and then extract it,
-        // but we really should do this:
-        //let mut ar = tar::Archive::new(copyreader);
-        let mut buf = vec![];
-        copyreader
-            .read_to_end(&mut buf)
-            .with_context(|| "reading base backup from page server failed")?;
-        let mut ar = tar::Archive::new(buf.as_slice());
-        ar.unpack(&pgdata)
-            .with_context(|| "extracting page backup failed")?;
+        if config_only {
+            //Just create an empty config file
```

Comment:
I don't know why `cargo fmt` doesn't handle this, but I would add a space after `\\`